lib/load.ex

```diff
@@ -25,6 +29,13 @@ defmodule Load do
     end)
   end
 
+  def stop() do
+    DynamicSupervisor.which_children(Load.Worker.Supervisor)
+    |> Enum.each(fn {_id, pid, :worker, _} ->
+      send(pid, :exit)
+    end)
+  end
```

Comment:
I think this only stops local children, I would rather go through scale down to implement stop as this has an effect only on the local node.

Reply:
please implement via scale(0)

Reply:
def stop, do: scale(0)

Reply:
Why not just stop supervisor. This should stop all children as well.